hi-ml/src/health_ml/runner.py

```diff
@@ -147,6 +147,9 @@ def parse_and_load_model(self) -> ParserResult:
         apply_overrides(container, parser2_result.overrides)  # type: ignore
         container.validate()
 
+        # Allow overriding ExperimentConfig params from within the container
+        container.update_experiment_config(experiment_config)
```

Comment:
I think command line should overwrite what is the config file, therefore this should happen before `apply_overrides`